pkg/pillar/vendor/github.com/docker/distribution/registry/api/errcode/errors.go

```diff
@@ -207,11 +207,11 @@ func (errs Errors) MarshalJSON() ([]byte, error) {
 	for _, daErr := range errs {
 		var err Error
 
-		switch daErr.(type) {
```

Comment:
Let's not mix up PRs. If this is not relevant to what we are doing, put it in a separate PR, please.

Reply:
Yeah -- lets remove it @adarsh-zededa

Reply:
These changes were generated as part of go.mod changes.